beat/models/seismic.py

```diff
@@ -591,7 +591,10 @@ def compute_var_reduction(data_trc, weight, result):
         var_reds = OrderedDict()
 
         for i in range(ndatasets):
-            nslc_id = utility.list2string(self.datasets[i].nslc_id)
+            if results[i].spectra:
+                nslc_id = utility.list2string(self.datasets[i].nslc_id+('spc',))
```

Comment:
please dont touch nslc_id, it has many implications together with snuffler. with the spectrum flag at dataset and the spectraltarget, this would become obsolete